src/utils/ucc_coll_utils.c

```diff
@@ -224,11 +238,27 @@ ucc_ep_map_t ucc_ep_map_from_array(ucc_rank_t **array, ucc_rank_t size,
     } else {
         map.type            = UCC_EP_MAP_ARRAY;
         map.array.map       = (void *)(*array);
-        map.array.elem_size = sizeof(ucc_rank_t);
+        map.array.elem_size = elem_size;
```

Comment:
do you  need to pass elem_size to that function?
why not
```
map.array.elem_size = is64 ? sizeof(uint64_t): sizeof(ucc_rank_t)
```

Reply:
you are correct. changed.